pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "fhir.resources",
     "antlr4-python3-runtime",
     "antlr-ast",
-    "antlr-tools",
+#    "antlr-tools",
```

Comment:
I think I need some help here. (Sorry, I am not very proficient with Python!)  Every time I try to build the project (e.g. `pip install .`) with this dependency included, I get this error:

```
5.654 ERROR: Could not find a version that satisfies the requirement antlr-tools (from tricc-oo) (from versions: none)
5.655 ERROR: No matching distribution found for antlr-tools
```

Am I missing something to get this to work?  (You should be able to reproduce the error by re-including this package and then trying to build the Dockerfile...)